integration/dkg/dkg_emulator_suite.go

```diff
@@ -397,31 +404,37 @@ func (s *EmulatorSuite) isDKGCompleted() bool {
 	return bool(value.(cadence.Bool))
 }
 
-func (s *EmulatorSuite) getResult() []string {
-	script := fmt.Sprintf(`
-	import FlowDKG from 0x%s
-
-	access(all) fun main(): [String?]? {
-		return FlowDKG.dkgCompleted()
-	} `,
-		s.env.DkgAddress,
-	)
-
-	res := s.executeScript([]byte(script), nil)
+func (s *EmulatorSuite) getResult() (crypto.PublicKey, []crypto.PublicKey, flow.DKGIndexMap) {
```

Comment:
similarly here: 
* we are retrieving DKG setup parameters (`flow.DKGIndexMap`) and the DKG result
* DKG setup parameters are chronologically first, so I would suggest to reflect that in the output order



```suggestion
// getResult retrieves the DKG setup parameters (`flow.DKGIndexMap`) and the DKG result from the DKG white-board smart contract.
func (s *EmulatorSuite) getParametersAndResult() (flow.DKGIndexMap, crypto.PublicKey, []crypto.PublicKey) {
```